stage-ovos/02-ovos/files/mycroft.conf

```diff
@@ -1,66 +1,68 @@
-         },
-      "retry_mic_init" : false,
-      "wake_word_upload": {
-         "disable": true}
-      },
-   "precise": {
-      "use_precise": false
-      },
-   "hotwords": {
-      "hey_mycroft-precise_lite": {
-         "module": "ovos-ww-plugin-precise-lite",
-         "model": "/home/ovos/.local/share/precise-lite/hey_mycroft.tflite",
-         "listen": true,
-         "sound": "snd/start_listening.wav",
-         "expected_duration": 3,
-         "trigger_level": 3,
-         "sensitivity": 0.5,
-         "fallback_ww": "hey_mycroft_openwakeword"
-      },
-      "hey_mycroft_openwakeword": {
-         "module": "ovos-ww-plugin-openwakeword",
-         "models": ["home/ovos/.local/share/openwakeword/hey_mycroft_v0.1.onnx"],
-         "threshold": 0.3,
-         "custom_verifier_threshold": 0.1
-         },
-      "hey_mycroft_vosk": {
-         "module": "ovos-ww-plugin-vosk",
-         "model_folder": "home/ovos/.local/share/vosk/vosk-model-small-en-us-0.15",
-         "samples": ["hey mycroft", "hey microsoft", "hey mike roft", "hey minecraft"],
-         "rule": "fuzzy",
-         "listen": true,
-         "fallback_ww": "hey_mycroft_pocketsphinx"
-         },
-      "hey_mycroft_pocketsphinx": {
-         "module": "ovos-ww-plugin-pocketsphinx",
-         "phonemes": "HH EY . M AY K R AO F T",
-         "threshold": 1e-90,
-         "lang": "en-us",
-         "listen": true
-         }
-      },
-   "stt": {
-      "fallback_module": ""
-      }
+    "ready_settings": ["skills"],
+    "confirm_listening": true,
+    "play_wav_cmdline": "paplay %1",
+    "play_mp3_cmdline": "mpg123 %1",
+    "ipc_path": "/ramdisk/mycroft/ipc/",
+    "network_tests": {
+        "ip_url": "https://api.ipify.org",
+        "dns_primary": "1.1.1.1",
+        "dns_secondary": "8.8.8.8",
+        "web_url": "http://nmcheck.gnome.org/check_network_status.txt",
+        "web_url_secondary": "https://checkonline.home-assistant.io/online.txt",
+        "captive_portal_url": "http://nmcheck.gnome.org/check_network_status.txt",
+        "captive_portal_text": "NetworkManager is online"
+        },
+    "listener": {
+        "mute_during_output": false,
+        "instant_listen": true,
+        "VAD": {
+            "module": "ovos-vad-plugin-silero",
+            "ovos-vad-plugin-silero": {"threshold": 0.2},
+            "ovos-vad-plugin-webrtcvad": {"vad_mode": 3}
+            },
+        "retry_mic_init" : false,
+        "wake_word_upload": {
+            "disable": true}
+    },
+    "precise": {
+        "use_precise": false
+        },
+    "hotwords": {
+        "hey_mycroft": {
+            "module": "ovos-ww-plugin-openwakeword",
+            "models": ["/home/ovos/.local/share/openwakeword/hey_mycroft_v0.1.onnx"],
+            "threshold": 0.4
+            },
+        "hey_mycroft_vosk": {
+            "module": "ovos-ww-plugin-vosk",
+            "model_folder": "/home/ovos/.local/share/vosk/vosk-model-small-en-us-0.15",
```

Comment:
Same as above.  Let the plugin download a model

Reply:
Does this mean that when pip loads a plugin that the plugin will edit mycroft.conf? Is that the way things work now?

Reply:
you need to remove `"model_folder" here, and it will download what is needed

Reply:
no, the plugin will auto download the default model if this value is not set, by setting the model path you are making the plugin skip the download step. 

makes sense if the image already includes the downloaded model and allows for the device to just work without internet, otherwise connection is needed for initial download, but even then the config should not be set and just let the plugin find the model as already downloaded, only the user should change this path if he manually downloaded something

Reply:
Got it. Thank you!

Reply:
I believe this is working now.